test/integ/separableCompilation/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 #
 
-if((NOT ALPAKA_ACC_GPU_CUDA_ENABLE AND NOT ALPAKA_ACC_GPU_HIP_ENABLE) OR
+if((NOT ALPAKA_ACC_GPU_CUDA_ENABLE AND NOT ALPAKA_ACC_GPU_HIP_ENABLE AND NOT ALPAKA_ACC_ANY_BT_OMP5_ENABLE ) OR
```

Comment:
separable compilation is not supported? Is this also the known linker error that will be fixed with gcc-11?

Reply:
The issue here is that functions which are not defined inline have to be enclosed in
```c++
#pragma omp declare target
...
#pragma omp end declare target
```
so targe code is generate and it can be called  in target regions. This cannot be worked into the `ALPAKA_FN_HOST_DEVICE` macro. Note, even if there would only be one pragma in from required, like in OpenACC, we could still not map this to the macro, because this would break template declarations:
```c++
template<class T>
ALPAKA_FN_DEVICE void f(int);
```
would expand to
```c++
template<class T>
_Pragma("acc routine seq") void f(int);
```
but you cannot place the pragma between the template and the function part.

Reply:
Is this missing support for separable compilation documented somewhere?

Reply:
No, I only disabled the test. Which would be the place to document this?

Reply:
It should probably be part of `README_OMP5.md`

Reply:
Added: https://github.com/alpaka-group/alpaka/pull/1126/commits/9e78aa35b9138cc5a096d5e597833554c9662623
(also some updates on compiler status)